torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Callable, Union, Dict
+from torch.fx.graph_module import GraphModule
+from torch.fx.node import Node
+from torch.fx.graph import Graph
+
+Target = Union[Callable[..., Any], str]
+
+def replace_target_nodes_with(
+    fx_module: GraphModule,
+    old_op: str,
+    old_target: Target,
+    new_op: str,
+    new_target: Target,
+):
+    """Modifies all nodes in fx_module.graph.nodes which match the specified op code and target,
+    and updates them to match the new op code and target"""
+    new_graph = Graph()
+    val_map : Dict[Node, Node] = {}
+    for node in fx_module.graph.nodes:
+        if node.op == old_op and node.target == old_target:
+            new_args = tuple([val_map[arg] for arg in node.args])
+            val_map[node] = new_graph.create_node(new_op, new_target, new_args, node.kwargs, node.name)
+        else:
+            val_map[node] = new_graph.node_copy(node, lambda n : val_map[n])
+    new_graph.output(new_graph.nodes[-1])
```

Comment:
The output of a graph may not simply be a reference to the last node and similarly to `args` and `kwargs`, the output of a graph may be a container, so use `map_args`:

```
new_graph.output(torch.fx.graph.map_args(fx_module.graph.result, lambda n: val_map[n]))
```